packages/testing/src/code-matchers.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-useless-escape */
+import { noIdents } from '@wixc3/common';
+import * as esTreePlugin from 'prettier/plugins/estree';
+import * as parserTypeScript from 'prettier/plugins/typescript';
+import { format } from 'prettier/standalone';
+
+type ChaiPluginMethod = (this: Chai.AssertionStatic, ...args: unknown[]) => unknown;
+
+const prettify: (code: string) => Promise<string> = (code) =>
+    format(code, {
+        parser: 'typescript',
+        plugins: [esTreePlugin, parserTypeScript],
+        endOfLine: 'lf',
+        singleQuote: true,
+    });
+
+const regex = /(\/\*)([^\/\*]*)(\*\/)/g;
+
+const mapper = (str: string, idx: number) => {
+    if (idx % 4 === 2) {
```

Comment:
Optional: check the negative condition first to reduce indentation, like this:

```
if (idx % 4 !== 2) return str;

return str.split...
```

Reply:
This code is lifted from Codux, I prefer not to change it where possible (just to make the change more frictionless)